.travis.yml

```diff
@@ -1,17 +1,68 @@
+sudo: false
 language: cpp
 
 compiler:
   - gcc
-  - clang
 
 env:
+  matrix:
+    - FUZZ=1
+    - NO_INLINE=1
+    - NO_BIT=1
+    - STATIC=1 NO_DEBUG=1
+    - SAFETY_PROFILE=1
+    - MOD_DEFAULT_FEATURES=1 USES_WCHAR=1
   global:
-    - ACE_ROOT=$TRAVIS_BUILD_DIR/ext/ACE_TAO/ACE
-    - TAO_ROOT=$TRAVIS_BUILD_DIR/ext/ACE_TAO/TAO
+    - ACE_ROOT=$TRAVIS_BUILD_DIR/ACE_TAO/ACE
+    - TAO_ROOT=$TRAVIS_BUILD_DIR/ACE_TAO/TAO
     - DDS_ROOT=$TRAVIS_BUILD_DIR
     - MPC_ROOT=$TRAVIS_BUILD_DIR/ext/MPC
     - LD_LIBRARY_PATH=$ACE_ROOT/lib:$DDS_ROOT/lib:$LD_LIBRARY_PATH
 
+matrix:
+  include:
+    - compiler: gcc
+      addons:
+        apt:
+          packages:
+            - openjdk-7-jdk
+      env: FUZZ=1 USE_JAVA=1
```

Comment:
Set this up before confirming that newer compilers were going to work.  Was also under the impression we were trying to somewhat mirror builds we maintain so that we could confirm when it was merged into the main branch we might see test failures on those builds, but at least builds should work.